src/main/resources/script_templates/Intro/Widgets.bsh

```diff
@@ -65,7 +66,7 @@ sb.append("Widgets BeanShell results:\n");
 sb.append("\n");
 sb.append("\tboolean = " + pBoolean + "\n");
 sb.append("\tbyte = " + pByte + "\n");
-sb.append("\tchar = " + "'" + pChar + "' [" + (int) pChar.charAt(0) + "]\n");
```

Comment:
The need for the `.charAt(0)` has been _inconsistent_ in the past, at least with the JavaScript version of the widget demo. I was always too lazy to track down why—in which circumstances it must be used, and in which circumstances it fails to work. My guess was Rhino vs. Nashorn. With BeanShell though, you'd expect that `.charAt` would be consistently incorrect.